server/migrations/20181029191536_plants.js

```diff
@@ -0,0 +1,29 @@
+exports.up = (knex, Promise) => {
+  knex.schema.createTable('plants', (table) => {
+    table.increments('id').primary();
+
+    table.string('plant_name');
+    table.string('status');
+    table.string('description');
+    table.string('genus');
+    table.string('species');
+    table.string('common_name');
+    table.string('hawaiian_name');
+    table.string('scientific_name');
+    table.text('story');
+    table.text('use');
+    table.enu('endangered',
```

Comment:
This is good but we actually don't have any endangered species at the moment; I think you'll need to add a "not endangered" category (not sure what best to call it).